src/compiler.adoc

```diff
@@ -1073,16 +1073,16 @@ first test fails as it did before, but our leap year test passes as expected.
 
 ==== Async Testing
 
-One of the peculiarities of _ClojureScript_ is that is built up on asynchronous, single
-theaded execution environments, and this has its own challenges.
+One of the peculiarities of _ClojureScript_ is that it runs on asynchronous,
+single-threaded execution environments, and this has its own challenges.
 
-In async execution environments we should be able test asynchronous functions. So,
-in this case, the _ClojureScript_ testing library offers a simply facility to do it:
-the `async` macro. It allows you to create blocks that play well with asynchronous code.
+In async execution environments, we should be able to test asynchronous
+functions.  To this end, the _ClojureScript_ testing library offers the `async`
+macro, allowing you to create tests that play well with asynchronous code.
```

Comment:
- simplify: In this case ... simple facility to do it -> To this end ...
- clarify: create blocks -> create tests